docs/docker/README.md

```diff
@@ -31,8 +33,18 @@ $ docker pull mcr.microsoft.com/playwright:bionic
 
 ### Run the image
 
+By default, the Docker image will use the `root` user to run the browsers. This will disable the Chromium sandbox which is not available with root. If you run trusted code (e.g. End-to-end tests), then the root user is in most cases fine. For web scraping or crawling, we recommend to create a separate user inside the Docker container and use the seccomp profile.
+
+#### Trusted environments
```

Comment:
Is "trusted environments" the right phrase? Environments sounds like CI/VMs that you run playwright on, but what we mean by this is trusted websites.

With the goal of making headings instantly relevant, I think we can title this as "End-to-end tests" and add a one-line description to what that means in this context. Similarly, the other heading can be "Crawling and scraping", followed by a one-line description of why sandboxing is important in that scenario.

Reply:
good catch! I updated it, let me know if it's fine for you.